### ISSUE TITLE
Generating of shortcuts doesn't work as expected

### ISSUE THREAD

**Issue by t0msk** [PhoenicisOrg/phoenicis#1544]
I created my own (custom via local installer) container and I installed BattleNET client and Hearthstone here, it works, but it didn't create a shortcuts.

When I open container in file browser I see  `lnk` files, like `Battle.net.lnk` and `Hearthstone.lnk` here. Why Phoenicis cannot notice it and show some dialog with shortcut suggestions.

**Comment by plata** [comment]
So you mean you installed StarCraft and Hearthstone from Battle.NET client? In this case, the behavior is as intended. Phoenicis cannot know that you installed something in Battle.NET.
If you want to have this behavior, we need a Battle.NET QuickScript (like for Steam).

**Comment by t0msk** [comment]
Yes and I think Phoenicis should know it, because these shortcuts (lnk files) are created in root directory of container.

So there chould be button like "Look for shortcuts" and if I click on it, then Phoenicis will look into root directory of container and grab all lnk files and show suggestions to user, PoL 4 does something like this, and I quite like it.

**Comment by plata** [comment]
In this case, I would say this is a duplicate of #1443.

**Comment by t0msk** [comment]
@plata Yes, but you said that I should open new issue :)
